### PR TITLE
s/currentPathElemenID/currentPathElementID

### PR DESCRIPTION
**Pull request by dcousens**
As mentioned in https://github.com/facebook/react/pull/4336

### CODE REVIEW

File: src/renderers/dom/client/ReactEventListener.js [hunk 1]
```diff
@@ -126,10 +126,10 @@ function handleTopLevelWithPath(bookKeeping) {
       );
 
       // Jump to the root of this React render tree
-      while (currentPathElemenID !== newRootID) {
+      while (currentPathElementID !== newRootID) {
```
Comment (dcousens) [inline src/renderers/dom/client/ReactEventListener.js]: `SyntaxError: ReactEventListener: Unexpected character ''` here as seen by https://travis-ci.org/facebook/react/jobs/70337892.

I did it via GitHub, maybe something weird snuck in.

Comment (zpao) [inline src/renderers/dom/client/ReactEventListener.js]: Looks like something did sneak in. I'll clean it up and merge.

screen shot 2015-07-09 at 11 48 31 pm

**Comment by zpao** [comment]
Thanks!

**Comment by dcousens** [comment]
Cheers @zpao,  sorry about the `^A`,  was a literal find replace using the GitHub UI.

**Pull request merged**